lib/aur-sync

```diff
@@ -201,11 +201,6 @@ tmp_view=$(mktemp -d --tmpdir "aurutils-$argv0-view.XXXXXXXX") || exit
 
 trap 'trap_exit' EXIT
 
-# Append remaining options to the aur-build command-line
-if (( $# )); then
-    build_extra_args+=("$@")
```

Comment:
that variable can be entirely removed from the file

Reply:
Done